src/pages/claim/ticket/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Center, Flex, Heading, VStack } from '@chakra-ui/react';
```

Comment:
Based on the design, i think this component logic should be in `[id].tsx`? 
I think they're on same page 🤔 

user receive a link `keypom.xyz/claim/ticket/123` -> backend receive data (sign up info, attendance gift) -> component show sign up info then *next page* should be either token gift or POAP nft gift

![image](https://user-images.githubusercontent.com/19740800/212023134-cded2b5c-7083-4eee-85c6-f14f62635793.png)

Reply:
hmm, then should we change the one for `nft` and `token` as well? wdyt?

Reply:
yeah.. actually only need to change page file name to `[id].tsx` right? but can change those separately